ModularTegustation/tegu_traitor_code/velvet_fu.dm

```diff
@@ -0,0 +1,262 @@
+			H.adjustStaminaLoss(-30)
+			H.mind.martial_art.streak = "receding_stance"
+		else
+			owner.visible_message("<span class='danger'>[owner] stops moving back.</i></b>")
+			return
+
+/datum/martial_art/velvetfu/proc/receding_stance(mob/living/carbon/human/A)
+	A.mind.martial_art.streak = ""
+
+// Twisted Stance
+/datum/action/twisted_stance
+	name = "Twisted Stance - Regenerates a lot of stamina, deals brute damage."
+	icon_icon = 'ModularTegustation/Teguicons/teguicons.dmi'
+	button_icon_state = "twisted_stance"
+
+/datum/action/twisted_stance/Trigger(mob/living/M, mob/living/user)
+	if(owner.incapacitated())
+		to_chat(owner, "<span class='warning'>You can't do stances while incapacitated..</span>")
+		return
+	var/mob/living/carbon/human/H = owner
+	if(H.mind.martial_art.streak == "twisted_stance")
+		owner.visible_message("<span class='danger'>[owner] untwists himself.</i></b>")
+		H.mind.martial_art.streak = ""
+	else
+		if(do_after(H, 0.1 SECONDS))
+			owner.visible_message("<span class='danger'>[owner] suddenly twists and turns, what a strange stance!</span>", "<b>You twist and turn, your ultimate stance is done!</b>")
+			H.adjustStaminaLoss(-60)
+			H.apply_damage(16, BRUTE)
+			H.mind.martial_art.streak = "twisted_stance"
+		else
+			owner.visible_message("<span class='danger'>[owner] untwists himself.</i></b>")
+			return
+
+/datum/martial_art/velvetfu/proc/twisted_stance(mob/living/carbon/human/A)
+	A.mind.martial_art.streak = ""
+
+//Flying Axe Kick - Deals Brute and causes bleeding. Costs 20 Stamina.
+/datum/martial_art/velvetfu/proc/flyingAxekick(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!can_use(A))
+		return FALSE
+	log_combat(A, D, "flying kick (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] flying kicked [D], such skill!</span>", \
+					"<span class='userdanger'>Your neck is flying kicked by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You flying kick [D]!</span>")
+	A.adjustStaminaLoss(20)
+	D.apply_damage(10, BRUTE)
+	var/obj/item/bodypart/bodypart = pick(D.bodyparts)
+	var/datum/wound/slash/moderate/crit_wound = new
+	crit_wound.apply_wound(bodypart)
+	playsound(get_turf(A), 'sound/weapons/slice.ogg', 50, TRUE, -1)
+	return TRUE
+
+//Goat Headbutt - Deals Brute and Stuns, in exchange for causing Brute to the user. Costs 40 Stamina.
+/datum/martial_art/velvetfu/proc/goatHeadbutt(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!can_use(A))
+		return FALSE
+	log_combat(A, D, "goat headbutt (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] headbutted [D]!</span>", \
+					"<span class='userdanger'>You're headbutted by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You swiftly headbutt [D]!</span>")
+	D.apply_damage(8, BRUTE)
+	D.Stun(5 SECONDS)
```

Comment:
5 seconds hard-stun..? I'd make it 3-4 seconds knockdown instead.

Reply:
gives them time to do a receding stance if they need to, since it costed him a lot of stamina